tests/protobuf/test_register_rest_actions_proto.py

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+
+import unittest
+
+from opensearch_sdk_py.protobuf import RegisterRestActionsProto_pb2
+
+
+class TestRegisterRestActionsProto_pb2(unittest.TestCase):
+    def test_register_rest_actions(self) -> None:
+        actions = RegisterRestActionsProto_pb2.RegisterRestActions()
+        actions.identity.uniqueId = "test"
+        actions.restActions[:] = ["foo", "bar"]
+        actions.deprecatedRestActions[:] = ["baz", "qux"]
+        serialized_str = actions.SerializeToString()
+
+        parsed_actions = RegisterRestActionsProto_pb2.RegisterRestActions()
+        parsed_actions.ParseFromString(serialized_str)
+        self.assertEqual(parsed_actions.identity.uniqueId, actions.identity.uniqueId)
+        self.assertCountEqual(parsed_actions.restActions, actions.restActions)
+        for i in range(len(actions.restActions)):
```

Comment:
Does an `self.assertEqual` not work here?

Looks like there's another 1-liner possible, not sure I like it better.

```
all([a == b for a, b in zip(actual, expected)])
```

Reply:
I'm still new enough to Python that after an hour (not continuous) of fiddling with it I gave up.  I tried `assertListEqual` but the value returned from protobuf is not a list.  
```
>       self.assertListEqual(parsed_actions.restActions, actions.restActions)
E       AssertionError: First sequence is not a list: ['foo', 'bar']
```
I tried `unittest2` and `assertItemsEqual` but couldn't get it to work (but that may have been a maze of import dependencies I didn't have the patience to navigate).

I'm skeptical that we have a need to test generated code at all and wasn't motivated to do more than [dumb code](https://www.oracle.com/technical-resources/articles/javase/devinsight-1.html).

Reply:
It's cause they aren't lists, they are arrays.

https://github.com/opensearch-project/opensearch-sdk-py/pull/47